scalafix/build.sbt

```diff
@@ -12,40 +12,40 @@ inThisBuild(
         url("https://geirsson.com")
       )
     ),
-    scalaVersion := V.scala212,
-    addCompilerPlugin(scalafixSemanticdb),
-    scalacOptions ++= List(
-      "-Yrangepos"
-    ),
+    scalaVersion := V.scala213,
+    semanticdbEnabled := true,
+    semanticdbIncludeInJar := true,
```

Comment:
@bjaglin: sbt-scalafix doesnt configure semanticdbTargetRoots by default ? 
I can either includeInJar or add the `input/semanticdbTargetRoots` in the `scalafixTestkitInputClasspath`

Reply:
@mlachkar  sbt-scalafix forwards scalacOptions to scalafix so that it can infer where semanticdb files were produced and thus add them to the classpath.

https://github.com/scalacenter/scalafix/pull/1389 made it possible to force it but I don't remember when this would be required? If you encounter an issue maybe we should fix the inference mechanism?

Reply:
I had an issue, the targetroot wasnt infered correctly and end up adding includeInJar. 

I will check and try to understand in which case it doesn't work!